build.gradle

```diff
@@ -36,10 +36,10 @@ def depVersions = [
         mockito: '3.3.3',
 
         // : functionaltests
-        concordion: '2.2.0',
+        concordion: '1.5.1',
```

Comment:
Why do we downgrade here? Did not we say that issue is not related to concordion version?

Reply:
@JudeNiroshan could you check the todo section of the PR ?

Reply:
7202712

Reply:
what about concordionParallelRunExtension ?

Reply:
1e58a47